css/styles.css

```
@@ -495,12 +614,40 @@
     color: #000000;
   }
 
+  .footer-social-link:hover svg {
+    fill: rgba(104, 162, 202, 1);
+  }
+
+  .footer-social-link:focus svg{
+    fill: rgba(104, 162, 202, 1);
+    border: 3px solid rgba(131, 179, 211, 1);
+  }
+
+  .footer-social-link:active svg{
+    fill: rgba(104, 162, 202, 1);
+    opacity: 30%;
+  }
```

Опять не следишь за форматированием, и отступ от края лишний так и остался
```suggestion
.footer-social-link:focus svg {
  fill: rgba(104, 162, 202, 1);
  border: 3px solid rgba(131, 179, 211, 1);
}

.footer-social-link:active svg {
  fill: rgba(104, 162, 202, 1);
  opacity: 30%;
}
```